lib/web/controllers/users.go

```diff
@@ -0,0 +1,24 @@
+package controllers
+
+import (
+	"net/http"
+
+	"go-google-scraper-challenge/helpers/log"
+	"go-google-scraper-challenge/view"
+
+	"github.com/foolin/goview"
+	"github.com/gin-gonic/gin"
+)
+
+type UsersController struct {
+	BaseWebController
+}
+
+func (c UsersController) New(ctx *gin.Context) {
```

Comment:
I think we should use the pointer receiver here:

```suggestion
func (c *UsersController) New(ctx *gin.Context) {
```

We haven't had the convention on this one yet, but for the struct method, people prefer using pointer receiver
https://stackoverflow.com/questions/27775376/value-receiver-vs-pointer-receiver